ajax/libs/color/0.11.3/index.js

```diff
@@ -0,0 +1,450 @@
+/* MIT license */
+var clone = require('clone');
+var convert = require('color-convert');
+var string = require('color-string');
```

Comment:
I think it does.
according to the author said...
```
Using requireJS or something similar, yes -- to my knowledge none of the dependencies requires anything Node.js specific.
```

https://github.com/cdnjs/cdnjs/issues/8531#issuecomment-244194889